css/style.css

```diff
@@ -449,13 +967,13 @@ body {
     transition: .5s all;
   }
 
-  .heading__hamburger:hover span:nth-child(1) {
+/*  .heading__hamburger:hover span:nth-child(1) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5612618)